packages/plugins/i18n/admin/src/index.js

```diff
@@ -102,12 +102,12 @@ export default {
                 name: 'pluginOptions.i18n.localized',
                 description: {
                   id: getTrad('plugin.schema.i18n.localized.description-content-type'),
-                  defaultMessage: 'Allow you to have content in different locales',
+                  defaultMessage: 'Allows translating an entry into different languages',
```

Comment:
Apologies if this is too nit picky, but this translation string would make more sense to me
```suggestion
                  defaultMessage: 'Allows the translation of an entry into different languages',
```

Reply:
I won't even start arguing with you 😂 Does your translation feel consistent with the one from D&P?

Reply:
We are never too picky when it comes to copy. 

"languages" is fine to me. I'd keep "translating" however mostly for consistency with D/P :D